sql/or_tools/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 SET(LOCAL_FILES
-  _knapsack_0-1.sql
+  _knapsack.sql
+  _knapsack.sql
```

Comment:
The user facing function file goes without underscore.
you are including the same file twice